hoi/simulation/simulation_hois.py

```diff
@@ -0,0 +1,758 @@
+import numpy as np
+
+# import matplotlib.pyplot as plt
+
+###############################################################################
+###############################################################################
+#                                 SWITCHER SIMULATIONS
+###############################################################################
+###############################################################################
+
+
+def simul_hois(
+    amplitude=1,
+    target=False,
+    target_frites=False,
+    n_trials=1000,
+    n_nodes=12,
+    n_times=None,
+    data_type="array",
+    time_bump=None,
+    time_length_bump=None,
+    triplet_character=None,
+    triplet_character_with_beh=None,
+):
+    """
+    Simulates High Order Interactions (HOIs) with or without behavioral
+     information, depending on the 'target' parameter. The simulation
+     can be conducted for a specified number of trials and nodes,
+     with an optional time component.
+
+    Parameters:
```

Comment:
For the documentation, can you please follow the Numpydoc convention? For [example](https://github.com/brainets/hoi/blob/main/hoi/utils/stats.py#L14)

```python
One line description (<79 chars).

Several lines description.

Parameters
-----------------
x: type
    Description
y: type
    Description

Returns
-----------
o: type
    Description
p: type
    Description
```

And same for other functions.